src/main/java/racing/Report.java

```diff
@@ -23,7 +24,7 @@ public List<Car> getCars() {
         return cars;
     }
 
-    public static List<String> findChampions(List<Car> cars) {
+    public List<String> findChampions() {
```

Comment:
```suggestion
    public List<Car> findChampions() {
```

우승자 자동차를 반환하는 것이 확장성 측면에서 더 좋지않을까요?
향후 우승자의 이름, 상태, 나이 등등 출력이 요구된다면
해당 메소드는 변경이 없고 view 영역에서만 가공처리하면 되니까요 😄